Performance.md

```diff
@@ -1,208 +1,212 @@
-For a program `Pgm` in a programming language `PL`, we use the
-[K framework](https://kframework.org) to generate
-a logical proof `PI(Pgm, PL)` that shows the correctness of an execution
-trace of `Pgm` using directly a formal semantics of `PL`.
-Such a logical proof can be automatically checked by a logical proof checker
-`proof_check(PI(Pgm, PL))`.
-Then, we generate a ZK proof that shows the correctness of
-an execution trace of `proof_check`.
-In other words, we generate a ZK proof that shows the correctness
-of a logical proof that shows the correctness of `Pgm` written in language `PL`.
-Thus, we call this benchmark set Proofs of Proofs, as we generate
-(ZK) proofs of (logical) proofs.
+This category combines ZK proofs and logical/mathematical proofs.
+For a program `Pgm` in a programming language `PL`, we use the [K framework](https://kframework.org) to generate a logical proof `PI(Pgm, PL)`. This proof demonstrates the correctness of an execution trace of `Pgm` using a formal semantics of `PL`.
+
+A logical proof checker can automatically verify `proof_check(PI(Pgm, PL))`. We then generate a ZK proof demonstrating the correctness of an execution trace of `proof_check`.
+
+In essence, we generate a ZK proof that validates the correctness of a logical proof, which in turn verifies the correctness of `Pgm` written in the language `PL`. This is why we refer to this benchmark set as Proofs of Proofs--it generates (ZK) proofs of (logical) proofs.
+
+## ZK Backend Evaluation
+This describes the ZK backends we evaluated, including details on their performance.
 
 ### ZK Backends
 
-We consider the following ZK backends:
+The ZK backends under consideration are:
 - [Cairo](https://www.cairo-lang.org/)
 - [Lurk](https://lurk-lang.org/)
 - [RISC Zero](https://www.risczero.com/)
 - [zkLLVM](https://github.com/NilFoundation/zkLLVM)
 
 ## Performance Tables
 
-- machine spec: AMD Ryzen 9 7950X(16 cores/32 threads/128GB), 4090RTX
-- memory swap: 108GB
-- performance time measured in seconds
-- besides RISC Zero, all other implementations were executed using the diff between
-  the timestamps of the start and end of each possible execution phase.
-- RISC Zero has its own performance counter, so we use it to measure the execution time and cycles.
-- "CPU" prefix = without GPU acceleration
-- "GPU" prefix = with GPU acceleration
-- last update date: Dec 19th, 2023.
+- Machine Spec: AMD Ryzen 9 7950X(16 cores/32 threads/128GB), 4090RTX
+- Memory Swap: 108 GB
+- Performance Time Measured in Seconds
+- The execution time for all implementations, except RISC Zero, was measured by taking the difference between the start and end timestamps for each execution phase.
+- RISC Zero has its own performance counter, which is used to measure the execution time and cycles.
+- The "CPU" prefix denotes implementation **without** GPU acceleration
+- The "GPU" prefix denotes implementation **with** GPU acceleration
+- Last Update: Dec 19th, 2023
```

Comment:
There are many parts in this benchmark set and they have different "Last Update" dates. Either we change the date here to be the latest update date, or we simply remove it from here.